### DISCUSSION TITLE
Account suspended and immediately reinstated??

### DISCUSSION THREAD

**Discussion by pooksh** [community/community#116566] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was doing normal GitHub things, creating a new repo from my school's template when I was suddenly suspended on this account. I submitted a ticket in confusion, then I tried to reset my password again. Then I was able to reset it and login again. Weirdest account experience I've ever had. Should I just wait for a response to the ticket to explain what just happened or submit another one? I don't want to be suddenly locked out of here because I have group projects to work on.

**Comment by pooksh** [comment]
Forgot to mention that this is free account unlike many of the people who experienced this issue and I received no emails about the suspension when it happened.

**Comment by arvindrag** [comment]
suspended, not reinstated and now I have 'too many attempts to log in'

**Reply by pooksh** [reply]
try this link while signed out of any account https://support.github.com/request/landing and click can't sign in, submit a ticket and then try resetting the password? this is what i did exactly. it may not work since you have had too many attempts to log in, or we might just be experiencing a slightly different issue

**Reply by pooksh** [reply]
you might just need to wait a bit before attempting a password reset, it looks like some people are just silently getting their accounts back by resetting their password like @paulaustin-automutatio here: https://github.com/orgs/community/discussions/116569. very strange situation

**Reply by gpereiradev** [reply]
try logging in again, it worked

**Comment by github-staff** [comment, marked as answer]
Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated.

We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

**Reply by paulaustin-automutatio** [reply]
I hope you are going to send an individual email to each affected account to explain what went on. Customer service didn't respond to any emails.

**Reply by gensart-x** [reply]
I experienced the same issue with @paulaustin-automutatio , I've created a ticket last week but still no respond, on my account https://github.com/gensart-ai . Please. I really need it for work.

**Reply by tarunmm** [reply]
i too experienced the same issue with my account @tarunmanoharan of getting suspended(as of violating terms and conditions). i just want to know how fast it would be rectified as my both individual and group projects are present in it. i have create a ticket regarding the same but  didn't get reply yet.Kindly oversee it as soon as possible.Thanks in advance!!

**Reply by DebiduttaPanigrahi** [reply]
My account DUTTA-78150 encountered a similar problem, being suspended for breaching terms and conditions. I'm concerned about the timeframe for resolution, given that both my personal and collaborative projects rely on it. Although I've submitted a ticket, I'm still awaiting a response. Please prioritize this matter for rectification.

**Reply by alanzimm85** [reply]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).

**Reply by gbolahan8** [reply]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing to inform you that I am currently unable to log in to my GitHub account with the username "kinsomaz." Additionally, I am unable to request a password change, and I have not received any communication explaining the issue. I have attempted to contact support multiple times over the past few days, but I have not received any response.

This issue is causing significant disruption to my work, as I rely heavily on GitHub for my ongoing projects. I urgently need access to my account to continue my work.

Please assist me in resolving this matter as soon as possible. I would appreciate any information you can provide regarding the status of my account and the steps I need to take to regain access.

Thank you for your prompt attention to this urgent matter. I look forward to your response.

Best regards,
[@kinsomaz]
[[kinsomaz87@gmail.com](mailto:kinsomaz87@gmail.com)]

**Reply by ghost** [reply]
I don’t know when @github-staff  will respond to me. I haven’t violated any of their terms and conditions, yet it’s been a month since my account (https://github.com/Rohangith1) was suspended without any email or notification. This has caused a significant disruption to my work, and I’ve lost access to all of my repositories. GitHub support hasn’t responded to my emails either. I’ve noticed that this has been happening to many users. I wasn’t expecting this from GitHub. I hope @github-staff sees this message and at least responds to let me know when my account will be reviewed and restored. I didn’t know that GitHub support could be this unresponsive.

**Reply by lightla** [reply]
I want my code, if not recover my account, please give me code @github-staff 

account: light-souler

**Reply by KR34T1V** [reply]
I don't thinks this was fixed, Account suspended.
I tried to ask for reinstatement, it has been 2 months no response.

I would use GitLab, but some platforms only support Github.

**Reply by MrAnkitPratap** [reply]
URGENT - Account Suspended Without Notice + Support Emails Blocked

Hi GitHub Community & Staff,

My account "frontendninjadev" (pratapguptaankit@gmail.com) was suspended on September 11, 2025 without any email notification.

CRITICAL ISSUE: Cannot contact support@github.com because:
- Gmail: Blocked due to organizational policy  
- Outlook: Blocked due to organizational policy
- Multiple email providers: All blocked

This is severely affecting my professional work as a web developer.

Account Details:
- Username: frontendninjadev
- Email: pratapguptaankit@gmail.com  
- Suspended: September 11, 2025
- No policy violations that I'm aware of

Can GitHub staff please help with account recovery?

Thanks for urgent assistance!

**Comment by tarunmm** [comment]
can u explain it briefly??? please

**Reply by gensart-x** [reply]
The thread starter just reopened and closed the discussion.

Based on @github-staff answer, I think the only thing we can do about the suspension is just to wait for an update of our ticket.

**Comment by tarunmm** [comment]
Ok, thanks for the update.


Let me know if your ticket gets updated dude!

**Comment by MrHeartHacker04** [comment]
Hello I am trying to get back my suspended account from a long time but I didn't recive any reply from GitHub team 
It's been very long time still your community haven't responded to me on my GitHub account reinstatement please look at this once and reply me something for assurance of that you are solving my issue
My account name MrAbhishekKhairnar with email abhishekhairnar04@gmail.com and the raised ticket is 2721360 this ticket is raised on April 9 2024

**Comment by FreedomForSe7en** [comment]
Urgent: Account Suspension Issue

Dear GitHub Support Team,

I hope this message finds you well. I am writing to seek urgent assistance regarding an issue I am experiencing with my GitHub account. My account has been suspended, and I am unable to push my repositories from Visual Studio Code to GitHub. This issue is significantly impacting my workflow and project timelines.

Account Details:

Username: FreedomForSeven
Email Address: freedomforseven@gmail.com
![Screenshot 2024-07-01 042217](https://github.com/community/community/assets/174294245/9560b966-85b2-458d-ab29-2ef9f3ba1d86)

Issue Description: When attempting to push my code to GitHub from Visual Studio Code, I receive the following error message: “Git: remote: Your account is suspended. Please visit https://support.github.com for more information.”

I have recently created this account and have ensured that I am not violating any of GitHub’s terms of service. Despite this, my account remains suspended, and I am unable to proceed with my work. I have also tried the following troubleshooting steps without success:

Cleared my Git credentials from the system’s credential manager.
Reconfigured my Git settings in Visual Studio Code.
Attempted to use an alternate GitHub account, but the issue persists.
Impact: This suspension is causing significant delays in my project, as I am unable to push updates and collaborate with my team. It is crucial for me to have access to my GitHub account to continue my work and meet project deadlines.

Request: I kindly request your assistance in resolving this issue as soon as possible. Please provide any information regarding the reason for the suspension and the steps I need to take to restore my account. If there are any specific actions required from my end, please let me know, and I will address them promptly.

Thank you for your attention to this matter. I appreciate your prompt assistance and look forward to resolving this issue swiftly.

Best regards,

**Reply by Olamilekan1048** [reply]
pls help me try to activate the account back and i have money on the account

**Comment by ANKeshri** [comment]
@github-staff my account https://github.com/ANKeshri has been flagged without any violation to terms and conditions . I am able to view my profile but others cant view my profile, contributions , issues and pull requests . my account was active till 2:35 am , 6 july . But in morning 404 error came when other person was viewing my profile and i was unable to see my contributions . Please look into the matter and restore my account as soon as possible as I am involved in gssoc and will not be able to do that .

**Comment by ghost** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing to inform you that my GitHub account (seerathussain) has been suspended, and I am unable to access my repositories or push any changes. When attempting to push changes to my repository (Final-Year-Assignment), I encounter a 403 Forbidden error with the message indicating that my account is suspended.

I kindly request your assistance in resolving this issue. Please let me know the reason for the suspension and the steps I need to take to reinstate my account. I rely heavily on GitHub for my ongoing projects, and the suspension is causing a significant disruption to my work.

Thank you for your prompt attention to this matter. I look forward to your response.

Best regards,

[@seerathussain]
[azain670@gmail.com] associated with GitHub

**Comment by gbolahan8** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing to inform you that I am currently unable to log in to my GitHub account with the username "kinsomaz." Additionally, I am unable to request a password change, and I have not received any communication explaining the issue. I have attempted to contact support multiple times over the past few days, but I have not received any response.

This issue is causing significant disruption to my work, as I rely heavily on GitHub for my ongoing projects. I urgently need access to my account to continue my work.

Please assist me in resolving this matter as soon as possible. I would appreciate any information you can provide regarding the status of my account and the steps I need to take to regain access.

Thank you for your prompt attention to this urgent matter. I look forward to your response.

Best regards,
[@kinsomaz]
[kinsomaz87@gmail.com]

**Reply by gensart-x** [reply]
i feel sorry for your account bro. i had the same issue too, since March 2024, having even no a single reply from all my emails to GitHub Support email until now.

i think all we can do for this is just passed it on and continuing with the new account.

**Comment by MrHeartHacker04** [comment]
Hello I am Abhishek  I saw your email and your problem regarding GitHub
account.

Few months ago my GitHub account was suspended due to GitHub privacy policy
changes

But after 6 to 7 months later I got a reply from GitHub and they said we
look into your account and we release suspension from it, it is fine.

And to resolve this issue you have to raise a ticket regarding GitHub
account suspension with proper detaila like gmail linked and user name with
apologies to GitHub if I broke some rule.. that's it.

Whenever your complete this procedure you cannot raise same issue ticket it
shows error
So fill every detail in description as possible
And wait for reply
I hope this will help you alot and keep patience till they reply you.

Best regards
Abhishek Khairnar

**Comment by Danish451** [comment]
**### Subject: Account Suspended and Immediately Reinstated – Request for Clarification**

_Hello,

I am writing on behalf of my friend, Vishal Rawat (GitHub ID: [Vicky2114](https://github.com/Vicky2114), Email ID: vr6994056@gmail.com), who recently experienced an issue where his GitHub account was suspended without any clear reason, and then it was reinstated shortly after.

Vishal is unsure why this suspension occurred, as he hasn't violated any terms or policies. We would greatly appreciate any clarification on the cause of the suspension to ensure his account remains in good standing.

Please let us know if any further action is required to prevent this from happening again.

Thank you for your time and support._

**Best regards,
Vishal's friend,
Danish Khan**

**Comment by ghost** [comment]
Title: GitHub Account Suspended Without Notice - Seeking Urgent Assistance

Hello GitHub Support Team,

@github-staff 
My GitHub account (https://github.com/Lut-Lat-Aung) was suddenly suspended without any prior notice or explanation, which has severely impacted my work. I rely heavily on GitHub for my projects, and the suspension of my account means that my repositories are no longer accessible. The email connected to this suspended account is lutlataung03@gmail.com

Here are the steps I've taken so far:

Submitted a Support Ticket: I submitted a support ticket through GitHub's support page today. 
Reinstatement Request #3141131

My entire work has been suspended.
My repositories are inaccessible.
The lack of communication from GitHub has left me in the dark about the reason for the suspension.

Request:

I am urgently seeking assistance to understand the reason behind the suspension of my account.
I need help to restore access to my repositories as soon as possible.
Additional Context:

I have reviewed GitHub's terms of service and community guidelines to ensure compliance.
My work is critical to my projects and future creations, and this suspension has caused significant disruption.
I would greatly appreciate any advice, support, or guidance from the GitHub community or staff to resolve this issue promptly.

Thank you for your understanding and assistance.

Best regards,
[Lut-Lat-Aung](https://github.com/Lut-Lat-Aung)

**Comment by code-pekings** [comment]
Title: GitHub Account Suspended Without Notice - Seeking Urgent Assistance

Hello GitHub Support Team,

@github-staff
My GitHub account (https://github.com/perryhong) was suddenly suspended without any prior notice or explanation, which has severely impacted my work. I rely heavily on GitHub for my projects, and the suspension of my account means that my repositories are no longer accessible. The email connected to this suspended account is hpqyp2017@163.com

Here are the steps I've taken so far:

Submitted a Support Ticket: I submitted a support ticket through GitHub's support page yesterday.
Reinstatement Request #3169338

My entire work has been suspended.
My repositories are inaccessible.
The lack of communication from GitHub has left me in the dark about the reason for the suspension.

Request:

I am urgently seeking assistance to understand the reason behind the suspension of my account.
I need help to restore access to my repositories as soon as possible.
Additional Context:

I have reviewed GitHub's terms of service and community guidelines to ensure compliance.
My work is critical to my projects and future creations, and this suspension has caused significant disruption.
I would greatly appreciate any advice, support, or guidance from the GitHub community or staff to resolve this issue promptly.

Thank you for your understanding and assistance.

Best regards,
[perryhong](https://github.com/perryhong)

**Comment by spoiicy** [comment]
Hi @github-staff
My [account](github.com/spoiicy) has wrongly been suspended by github. I've activated 2FA as well post which still my account remains suspended.
I'd raised a Reinstatement Request #3175496 but my [account](github.com/spoiicy) has still not been activated. 
This is hampering my work as well as my organisation.
Could you please fast-track this.
Would really appreciate your prompt action on this. Also let me know in case you want my assistance as well.

Thank You 
Akshit Maheshwary

**Comment by Dave-Senn** [comment]
@github-staff Hey GitHub, I'm waiting for a response to my request since Nov 13, 2024 please take a look at #3094630

**Comment by ghost** [comment]
Title: GitHub Account Suspended Without Notice - Seeking Urgent Assistance

Hello GitHub Support Team,

@github-staff
My GitHub account (https://github.com/kar137) was suddenly suspended without any prior notice or explanation, which has severely impacted my work. I rely heavily on GitHub for my projects, and the suspension of my account means that my repositories are no longer accessible. The email connected to this suspended account is [bistakaran89@gmail.com](mailto:bistakaran89@gmail.com)

Here are the steps I've taken so far:

Submitted a Support Ticket: I submitted a support ticket through GitHub's support page yesterday.
Reinstatement Request #3226981

My entire work has been suspended.
My repositories are inaccessible.
The lack of communication from GitHub has left me in the dark about the reason for the suspension.

Request:

I am urgently seeking assistance to understand the reason behind the suspension of my account.
I need help to restore access to my repositories as soon as possible.
Additional Context:

I have reviewed GitHub's terms of service and community guidelines to ensure compliance.
My work is critical to my projects and future creations, and this suspension has caused significant disruption.
I would greatly appreciate any advice, support, or guidance from the GitHub community or staff to resolve this issue promptly.

Thank you for your understanding and assistance.

Best regards,
Karan Bista

**Comment by ronaksubedi** [comment]
Dear GitHub Support Team,

I hope this message finds you well.

I am writing on behalf of my friend, Nirdesh Pokharel (GitHub Username: coderNirdesh, Email: [nirdeshpokhrel29@gmail.com](mailto:nirdeshpokhrel29@gmail.com)), who recently faced an unexpected suspension of his GitHub account. The account was subsequently reinstated; however, Nirdesh is unclear about the reasons behind this action.

He has reviewed GitHub's policies and believes he has not violated any terms. We kindly request your assistance in providing clarity on the cause of the suspension to ensure his account's compliance moving forward.

Additionally, if there are any steps Nirdesh should take to prevent a recurrence, please let us know.

Thank you for your time and support._ @github-staff 

Best regards,
Ronak Subedi

**Comment by vallari-sharma** [comment]
@github-staff

I have my job interviews from next week and my account (valls-25) associated with [vallarisharma25@gmail.com](mailto:vallarisharma25@gmail.com) has been suspended for a reason unbeknownst to me . I assure you I have not violated any policy.

I issued a ticket (3422317) , however , currently if I make any new Git accounts with a new email and try to commit my codes , it says that my account is suspended. There is no way through at all for me , no matter if I make a new Git account .

I plead to you to restore and unsuspend my account , because it has all my important repositories .

**Comment by HimiteCat** [comment]
Title: GitHub Account Suspended Without Notice - Seeking Urgent Assistance

Hello GitHub Support Team,

@github-staff
My GitHub account (https://github.com/pjy612) was suddenly suspended without any prior notice or explanation, which has severely impacted my work. I rely heavily on GitHub for my projects, and the suspension of my account means that my repositories are no longer accessible. The email connected to this suspended account is [pjy612@163.com](mailto:pjy612@163.com)

Here are the steps I've taken so far:

Submitted a Support Ticket: I submitted a support ticket through GitHub's support page today.
Reinstatement Request #3519721

My entire work has been suspended.
My repositories are inaccessible.
The lack of communication from GitHub has left me in the dark about the reason for the suspension.

Request:

I am urgently seeking assistance to understand the reason behind the suspension of my account.
I need help to restore access to my repositories as soon as possible.
Additional Context:

I have reviewed GitHub's terms of service and community guidelines to ensure compliance.
My work is critical to my projects and future creations, and this suspension has caused significant disruption.
I would greatly appreciate any advice, support, or guidance from the GitHub community or staff to resolve this issue promptly.

Thank you for your understanding and assistance.

Best regards,
[pjy612](https://github.com/pjy612)

**Reply by pjy612** [reply]
Appeal can still only be made through one's own account.

**Comment by aalremibe** [comment]
Dear [Support Team],

I’m truly desperate for your help. I’m a student in the middle of a very important project this semester, and today I suddenly couldn’t log into my account_mirograph. It says my account has been suspended. I tried to contact support with another account but its the third account that has been suspended. I dont know really whats going on with github. 

I tried to reach the suppport team using my main account to say mirographism@gmail.com, but I couldn’t receive the verification SMS — it keeps saying my phone number is incorrect.

Please, I beg you to help me restore access to my account - mirographism@gmail.com - as soon as possible. My project is at a critical stage, and I have no way to continue without it. I would be deeply grateful for any assistance you can provide.

Thank you so much for your understanding and kindness.

Sincerely,
Alireza Mirzabeh

**Comment by cscneolikhurd-creator** [comment]
@github-staff              I have a new account on HitHub, it has been suspended. I have a request to activate it.
Subject: Account Suspension Appeal - @nbishnoi050

Dear GitHub Support Team,

My GitHub account (@nbishnoi050) has been suspended. I believe this may be due to one of the following reasons:

1. I recently created multiple repositories quickly
2. I was testing GitHub Pages features
3. Possible false positive in automated system

I assure you that I have not violated any Terms of Service intentionally. I am a genuine developer working on a free image tools project for the community.

Please review my account and restore access. I am willing to comply with all GitHub policies.
[free-image-tools.com](https://@nbishnoi050.github.io/free-image-tools/)
Email {nbishnoi063@gmail.com}
Thank you for your understanding.

Sincerely,
Nisha Bishnoi

**Comment by Chisom007-ng** [comment]
Hi, [@github_staff](https://github.com/github-staff), my other account has been suspended for some reason.
First, the account was hidden with restrictions. I did raise an appeal then, and I got a response, asking for my github use.

I did answer the question, and then I got an email saying that the restrictions had been lifted. However in reality, these restrictions have not been lifted till today, as I am unable to comment on the ticket with a suspended account still.

Here is the [ticket](https://support.github.com/ticket/4218675).

I would be pleased if this were attended to quickly

**Comment by Birdmannn** [comment]
This is quite sad though
